src/test/e2e/helpers/test-helper.ts

```diff
@@ -52,7 +92,13 @@ async function createApp(
     };
 
     // TODO: use create from server-impl instead?
-    return { request, destroy, services, config };
+    return {
+        request,
+        destroy,
+        services,
+        config,
+        api: httpApis(request, config),
```

Comment:
I think I could also do
```suggestion
        ...httpApis(request, config),
```